doc/syntax/comments.rdoc

```diff
@@ -35,3 +35,188 @@ syntax error:
     Will not work
     =end
   end
+
+= Magic Comments
+
+While comments are typically ignored by Ruby, special "magic comments" contain
+directives that affect how the code is interpreted.
+
+Top-level magic comments must start on the first line, or on the second line if
+the first line looks like `#! shebang line`.
```

Comment:
`<tt>#! shebang line</tt>`
(Probably you know, but you can check the rendering with running local `rdoc NEWS.md doc -o tmp/rdoc` -- or without passing `NEWS.md doc`, but it will take longer)

Reply:
Thanks!